__tests__/authenticatedFetch/headers/HeadersUtils.spec.ts

```diff
@@ -39,24 +39,4 @@ describe("Headers interoperability function", () => {
       ["content-type", "text/turtle"]
     ]);
   });
-
-  it("supports Comunica-style headers structures", () => {
-    // This enables testing that an object similar to the prototypes passed by
-    // Comunica is supported.
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    const myHeaders: any = {};
-    myHeaders["accept"] = "application/json";
-    myHeaders["content-type"] = "text/turtle";
-    myHeaders["raw"] = (): Record<string, string> => {
-      return {
-        accept: "application/json",
-        "content-type": "text/turtle"
-      };
-    };
-    const flatHeaders = flattenHeaders(myHeaders);
-    expect(Object.entries(flatHeaders)).toEqual([
-      ["accept", "application/json"],
-      ["content-type", "text/turtle"]
-    ]);
-  });
```

Comment:
Shall we have a test though for `non-iterable Headers-like structure` or something similar? As it might not be uncommon (https://github.com/inrupt/solid-auth-fetcher/pull/177#issuecomment-642501972).

Reply:
That's a good point. Added in 29d8f44bc63aeaf91ebb1bd0b8ab48604311eb07